tests/__init__.py

```diff
@@ -327,6 +327,33 @@ def test_addons_env(self):
                 ("test", "-d", "auto/addons/crm/migrations"),
             )
 
+    # HACK https://github.com/itpp-labs/misc-addons/issues/1014
+    # TODO Remove decorator
+    @prerelease_skip
+    def test_addons_env_double(self):
+        """Test double addon reference in addons.yaml"""
+        common_tests = (
+            ("test", "-d", "custom/src/rma-old/rma"),
+            ("test", "!", "-d", "custom/src/rma-old/rma_sale"),
+            ("test", "-d", "custom/src/rma-new/rma"),
+            ("test", "!", "-d", "custom/src/rma-new/rma_sale"),
+        )
+        for sub_env in matrix():
```

Comment:
There must be some check about the `__manifest__.py` stuff that I forgot about: https://github.com/Tecnativa/doodba/pull/370/checks?check_run_id=1694934996#step:8:192

Please skip <10.0 versions

Reply:
Ok, I was thinking the same :smile: :+1: Thanks